internal/host/validator.go

```diff
@@ -1945,3 +1952,18 @@ func (v *validator) noIscsiNicBelongsToMachineCidr(c *validationContext) (Valida
 
 	return ValidationSuccess, "Network interface connected to iSCSI disk does not belong to machine network CIDRs"
 }
+
+// multipathiSCSI - Related to the noIscsiNicBelongsToMachineCidr validation. This is executed when the installation disk is a multipath disk serving as a holder for iSCSI drives.
+func (v *validator) multipathiSCSI(c *validationContext, installationDisk *models.Disk) (ValidationStatus, string) {
+	var status ValidationStatus
+	msg := ""
+	for _, disk := range c.inventory.Disks {
+		if disk.DriveType == models.DriveTypeISCSI && strings.Contains(disk.Holders, installationDisk.Name) {
+			status, msg = v.standaloneiSCSI(c, disk)
+			if status != ValidationSuccess {
+				return status, msg
```

Comment:
this validation will report only the error on the first disk that fails the validation.

I think it would be better to run the validation on each iSCSI disk, and report a generic error on the multipath disk saying that at least one iSCSI disk is not eligible.